<launch>
    <include file="$(find turtlebot3_slam)/launch/turtlebot3_slam.launch"/>
    <node pkg='hw1' name='field_nav' type='field_nav_real.py'>
        <param name='ignore_dist' value='0.3' type='double'/>
        <param name='dist_exp' value='2' type='double'/>
        <param name='forward_vec' value='0.75' type='double'/>
        <param name='max_ang_rate' value='1.5' type='double'/>
        <param name='vel_scale' value='0.75' type='double'/>
        <param name='ang_scale' value='0.2' type='double'/>
        <param name='fov' value='1.5707963267948966' type='double'/>
        <param name='lidar_topic' value='scan' type='str'/>
        <param name='zero_dist' value='0.1' type='double'>
        <param name='invert_theta' value='true' type='bool'/>
    </node>
</launch>